### ISSUE TITLE
Asking for the best result of the code

### ISSUE THREAD

**Issue by tourist-wwj** [gospodima/Extended-SimGNN#10]
I'm happy to get the similar MSE with your result shown in the picture and your code is pretty good. However, it's larger than the result in the thesis(1.189). I hope to discuss about the best running result of the code and the possibility of repeating the results in the thesis. Thanks!

**Comment by gospodima** [comment]
Thank you for the feedback.
I can already say that I couldn't reproduce exact result from the original paper. You can read more details in these issues: #5, #3.
Let me know if you have any further questions.

**Comment by tourist-wwj** [comment]
Thanks for your quick reply and your guide!

**Comment by tourist-wwj** [comment]
Moreover, it's noticeable that I changed your function(create_batches)in simgnn.py , here's my new code:
    def create_batches(self):
       
        self.training_graph = self.training_graphs.shuffle()
        batches = list(itertools.product(self.training_graph, self.training_graph))
        source_loader = DataLoader(
            batches
            ,
            batch_size=self.args.batch_size,
        )

        return source_loader
I can get 560*560 graph pairs with this change, and I get about 2.5 *10^-3 MSE similar with you.
